CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Fixed
 
 - Do not panic in `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc` if `MeterProvider` returns a `nil` instrument. (#4875)
+- OpenTelemetry Collector based example for `go.opentelemetry.io/contrib/samplers/jaegerremote`. (#4892)
```

Comment:
A `Fixed` entry would be a bugfix, which this isn't. How about moving it to the `Changed` section?
Also, how about something along the lines of:

> Updated jaeger version in `go.opentelemetry.io/contrib/samplers/jaegerremote/example` example. (#4892)

Reply:
Thank you for the feedback. I also had some doubts about where should I add it while adding it to changelog. Of course 👍  I am going to move it in a minute.